generator/generator_tests_support/test_mwm_builder.hpp

```diff
@@ -5,16 +5,11 @@
 #include "std/string.hpp"
 #include "std/unique_ptr.hpp"
 
-class Classificator;
 
-namespace feature
-{
-class FeaturesCollector;
-}
-namespace platform
-{
-class LocalCountryFile;
-}
+class Classificator;
+class FeatureBuilder1;
+namespace feature { class FeaturesCollector; }
```

Comment:
Please, return old style, when forward declarations are on individual lines, like it was. It's more common in the project.